noxfile.py

```diff
@@ -512,16 +519,34 @@ def prerelease(session: nox.sessions.Session, tests_path):
         "--upgrade",
         "pandas",
     )
-    session.install(
-        "--upgrade",
-        "-e",  # Use -e so that py.typed file is included.
-        "git+https://github.com/ibis-project/ibis.git#egg=ibis-framework",
-    )
+    already_installed.add("pandas")
+
+    # TODO(shobs):
+    # Commit https://github.com/ibis-project/ibis/commit/c20ba7feab6bdea6c299721310e04dbc10551cc2
+    # introduced breaking change that removed the following:
+    #   ibis.expr.rules.column
+    #   ibis.expr.rules.value
+    #   ibis.expr.rules.any
+    # Let's exclude ibis head from prerelease install list for now. Instead, use
+    # a working ibis-framework version resolved via setup.by (currently resolves
+    # to version 6.2.0 due to version requirement "6.2.0,<7.0.0dev").
+    # We should enable the head back once bigframes support a version that
+    # includes the above commit.
+    # session.install(
+    #    "--upgrade",
+    #    "-e",  # Use -e so that py.typed file is included.
+    #    "git+https://github.com/ibis-project/ibis.git#egg=ibis-framework",
+    # )
+    session.install("--no-deps", "ibis-framework==6.2.0")
+    already_installed.add("ibis-framework")
+
     # Workaround https://github.com/googleapis/python-db-dtypes-pandas/issues/178
     session.install("--no-deps", "db-dtypes")
+    already_installed.add("db-dtypes")
 
     # Workaround to install pandas-gbq >=0.15.0, which is required by test only.
```

Comment:
Let's test with the latest `google-cloud-bigquery` and `google-cloud-bigquery-storage`, too. I'd like to catch breaking changes in those packages before they get released and affect us.

```suggestion
    # Ensure we catch breaking changes in the client libraries early.
    session.install(
       "--upgrade",
       "-e",
       "git+https://github.com/googleapis/python-bigquery.git#egg=google-cloud-bigquery",
    )
    already_installed.add("google-cloud-bigquery")
    session.install(
       "--upgrade",
       "-e",
       "git+https://github.com/googleapis/python-bigquery-storage.git#egg=google-cloud-bigquery-storage",
    )
    already_installed.add("google-cloud-bigquery-storage")

    # Workaround to install pandas-gbq >=0.15.0, which is required by test only.
```